### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/response-status-code.md`

### PR DESCRIPTION
**Pull request by batlopes**
Added translation for pt to `docs/pt/docs/tutorial/response-status-code.md`

### CODE REVIEW

File: docs/pt/docs/tutorial/response-status-code.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+!!! note "Nota"
+    Observe que `status_code` é um parâmetro do método "decorador" (get, post, etc). Não da sua função de *operação de caminho*, como todos os parâmetros e corpo.
+
+O parâmetro `status_code` recebe um número com o código de status HTTP.
+
+!!! info "Informação"
+    `status_code` também pode receber um `IntEnum`, como o do Python <a href="https://docs.python.org/3/library/http.html#http.HTTPStatus" class="external-link" target="_blank">`http.HTTPStatus`</a>.
+
+Será:
```
Comment (julianofischer) [inline docs/pt/docs/tutorial/response-status-code.md]: Dessa forma:

File: docs/pt/docs/tutorial/response-status-code.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    Observe que `status_code` é um parâmetro do método "decorador" (get, post, etc). Não da sua função de *operação de caminho*, como todos os parâmetros e corpo.
+
+O parâmetro `status_code` recebe um número com o código de status HTTP.
+
+!!! info "Informação"
+    `status_code` também pode receber um `IntEnum`, como o do Python <a href="https://docs.python.org/3/library/http.html#http.HTTPStatus" class="external-link" target="_blank">`http.HTTPStatus`</a>.
+
+Será: 
+
+* Retorne esse código de status na resposta.
```
Comment (julianofischer) [inline docs/pt/docs/tutorial/response-status-code.md]: Este código de status será retornado na resposta.

File: docs/pt/docs/tutorial/response-status-code.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+O parâmetro `status_code` recebe um número com o código de status HTTP.
+
+!!! info "Informação"
+    `status_code` também pode receber um `IntEnum`, como o do Python <a href="https://docs.python.org/3/library/http.html#http.HTTPStatus" class="external-link" target="_blank">`http.HTTPStatus`</a>.
+
+Será: 
+
+* Retorne esse código de status na resposta. 
+* Documente-o como tal no esquema OpenAPI (e, portanto, nas interfaces do usuário):
```
Comment (julianofischer) [inline docs/pt/docs/tutorial/response-status-code.md:26]: Será documentado como tal no esquema OpenAPI (e, portanto, nas interfaces do usuário):

**Review by julianofischer — CHANGES_REQUESTED** [review]
Hi fellow.
In my opinion the text needs some modifications for clarity.

File: docs/pt/docs/tutorial/response-status-code.md (hunk 1 above)
Comment (batlopes) [reply docs/pt/docs/tutorial/response-status-code.md]: Done.

File: docs/pt/docs/tutorial/response-status-code.md (hunk 2 above)
Comment (batlopes) [reply docs/pt/docs/tutorial/response-status-code.md]: Done.

File: docs/pt/docs/tutorial/response-status-code.md (hunk 3 above)
Comment (batlopes) [reply docs/pt/docs/tutorial/response-status-code.md]: Done.

**Review by Wuerike — APPROVED** [review]

File: docs/pt/mkdocs.yml [hunk 4]
```diff
@@ -67,6 +67,8 @@ nav:
   - tutorial/extra-data-types.md
   - tutorial/query-params-str-validations.md
   - tutorial/cookie-params.md
+  - tutorial/header-params.md
```
Comment (lsglucas) [inline docs/pt/mkdocs.yml:70]: Nice job with the translation!
Why have you added this line to `mkdocs.yml`? As far as I know the header params page is not translated yet hence it should not be added to this file yet

Comment (batlopes) [reply docs/pt/mkdocs.yml]: sorry, I added it by mistake.

**Review by stlucasgarcia — APPROVED** [review]

**Review by jvmazagao — APPROVED** [review]

**Pull request merged**